toolkit/scripts/sodiff/mariner-sodiff.sh

```diff
@@ -86,6 +113,9 @@ echo "######################"
 if [[ $pkgsFound -gt 0 ]]; then
     echo "The Following Packages Are in Need of an Update:"
     cat "$sodiff_out_dir"/sodiff-summary.txt
+    if [[ "$sodiff_script_error" -eq "true" ]]; then
+        exit 1
+    fi 
```

Comment:
I'd suggest to always error out and leave it to the caller to deal with the error as they see fit.

Reply:
This optionality was introduced for use in the pipelines.  In case there is something wrong where we need to ignore the hard error in the check, this is a backstop.